Cargo.toml

```diff
@@ -78,3 +78,4 @@ testutils = { path = "lib/testutils" }
 default = []
 bench = ["criterion"]
 vendored-openssl = ["git2/vendored-openssl", "jujutsu-lib/vendored-openssl"]
+watchman = ["jujutsu-lib/watchman"]
```

Comment:
This is why I initially thought you compiled `watchman` into jj.

Perhaps rename the feature to `watchman_client` to reduce confusion? In the future, there could be a potential `watchman` feature that actually compiles `watchman` in.

If that doesn't seem to reduce confusion, you can leave it as-is.

Reply:
> Perhaps rename the feature to watchman_client to reduce confusion? In the future, there could be a potential watchman feature that actually compiles watchman in.

It is essentially impossible to compile Watchman as part of jj, unfortunately. I had a considerable amount of trouble compiling it from source last time I worked with it: I actually had to do my work in a Linux VM because I couldn't get it to compile on Mac. (Although maybe the Sapling folks have somehow figured out a way to bundle Watchman?)

Reply:
As I said in https://github.com/martinvonz/jj/pull/1731#issuecomment-1603588294, I confused it with `watchexec`, sorry.